lambda/matching.mli

```diff
@@ -52,3 +52,4 @@ val expand_stringswitch:
     lambda option -> lambda
 
 val inline_lazy_force : lambda -> scoped_location -> lambda
+val lazy_from_val : lambda -> scoped_location -> lambda
```

Comment:
This is a strange place to define such a function since it is not used inside `Matching`